heartbeat/LVM-activate

```diff
@@ -790,7 +790,7 @@ lvm_status() {
 	fi
 
 	if [ $dm_count -eq 0 ]; then
-		return $OCF_NOT_RUNNING
```

Comment:
We should probably do an `if ocf_is_probe; then` here to return NOT_RUNNING for probe-actions.

Reply:
ok，It's fixed, can you please check it?

Reply:
It's fixed, can you please check it?

Reply:
It should be reversed. ERR_GENERIC in the else-clause.

Reply:
I have some questions  ,doesn't  "ocf_is_probe " mean cluster is using monitor ?

Reply:
in commits  , i just reversed it before ⊙︿⊙

Reply:
> I have some questions ,doesn't "ocf_is_probe " mean cluster is using monitor ?

Correct. In this agent the monitor-action is in lvm_status(). Probe is a special monitor where it only checks whether it's running or not (e.g. for disabled resources).

Reply:
ok got it，It's fixed